instrumentation-api-incubator/src/main/jflex/SqlSanitizer.jflex

```diff
@@ -329,6 +391,27 @@ WHITESPACE          = [ \t\r\n]+
           appendCurrentFragment();
           if (isOverLimit()) return YYEOF;
       }
+  "CREATE" {
+          if (!insideComment) {
```

Comment:
similar to code below - extract method

Reply:
Can you help suggest what kind of extraction you are looking for? These are the condition checks for the sanitizer to decide whether it's a valid special token to handle. I don't see how to extract them into some base class.

Reply:
I think you can create a method and pass `new Create()` to it

Reply:
I'm uncertain if the modifications would be worthwhile. If we create a method and pass new `Create()` to it, it introduces redundancy by instantiating a new `Create` object every time the keyword matches, even if the  `CREATE` is `insideComment`. Conversely, with the current implementation, people will know, at the first glance, that the `Create` operation will only occur within an actual statement.

I understand that there's some duplication in the keyword handling, but the code structure is still easy to read and maintain. Unless we want a "BIG" refactoring here to compact all the similar duplication, I would prefer to keep it as is.